Java/sort/CountInversion.java

```diff
@@ -0,0 +1,46 @@
+import java.util.*;
+
+public class CountInversion{
+    private static int mergeSort(int[] arr, int low, int high){
+        int count = 0;
+        if(low < high){
+            int mid = (low+high)/2;
```

Comment:
@pranjalshinde Avoid this technique to find the middle element since this can cause an integer overflow. Use `int mid = low + ((high - low) / 2);`.

Reply:
@iamrajiv  I have made suggested changes.. Please review here #1931